pkgs/applications/science/misc/netlogo/default.nix

```diff
@@ -0,0 +1,43 @@
+{ jre, stdenv, fetchurl, makeWrapper }:
+
+
+stdenv.mkDerivation rec {
+  name = "NetLogo-${version}";
+  version = "6.0.4";
+
+
+  src = fetchurl {
+    url = "https://ccl.northwestern.edu/netlogo/6.0.4/NetLogo-6.0.4-64.tgz";
+    sha256 = "0dcd9df4dfb218826a74f9df42163fa588908a1dfe58864106936f8dfb76acec";
+  };
+
+
+  buildInputs = [ makeWrapper ];
+
+
+  installPhase = ''
+    mkdir -pv $out/share/netlogo
+    cp -rv app readme.md $out/share/netlogo
+
+    # launcher with `cd` is required b/c otherwise the model library isn't usable
```

Comment:
This can be simplified like this with `makeWrapper`

```
makeWrapper "${jre}/bin/java" "$out/bin/netlogo" \
      --run "cd $out/share/netlogo/app" \
      --add-flags "-jar netlogo-${version}.jar"
```

Reply:
Thank you!